### ISSUE TITLE
Show `mangage_externals` in get or run?

### ISSUE THREAD

**Issue by sunnivin** [MetOs-UiO/CTSM-Norway-Documentation#11]
Should we show the `[CTSM_ROOT]$ ./manage_externals/checkout_externals` in the documentation for [Get CTSM](https://metos-uio.github.io/CTSM-Norway-Documentation/get/) or [Run CTSM](https://metos-uio.github.io/CTSM-Norway-Documentation/run)? 

I think it can be very confusing for the beginner if we are not consistent.

**Comment by devarajun** [comment]
I would say for getting a complete CTSM model setup or more specifically extracts scripts for managing external source directories.

**Comment by ecaas** [comment]
I definitely think it should be in Get, and maybe add a note in Run on when you should rerun it? Also, I think it could be better explained what the command actually do, it is not necessarily clear to a beginner what an "external" is.

**Comment by sunnivin** [comment]
Would you agree to take on the task to more properly explain the `externals` step @ecaas? 

Currently, this is not consistent and described both in [get](https://metos-uio.github.io/CTSM-Norway-Documentation/get/) and [run](https://metos-uio.github.io/CTSM-Norway-Documentation/run/).

**Comment by ecaas** [comment]
Yes I can do that, but I don't have time before the end of the week to look at it.

**Comment by sunnivin** [comment]
> Yes I can do that, but I don't have time before the end of the week to look at it.

That's perfect!

**Comment by ecaas** [comment]
Hi all! in [get](https://metos-uio.github.io/CTSM-Norway-Documentation/get/), `[CTSM_ROOT]$ ./manage_externals/checkout_externals` is included under both the ["user"](https://metos-uio.github.io/CTSM-Norway-Documentation/get/#how-to-get-ctsm-for-users) header and the ["developer" ](https://metos-uio.github.io/CTSM-Norway-Documentation/get/#how-to-get-ctsm-for-developers )header. Do we want to keep it that way (meaning there will be some repetition), or is it better to create one new header for the manage externals stuff?

**Comment by sunnivin** [comment]
> Hi all! in [get](https://metos-uio.github.io/CTSM-Norway-Documentation/get/), `[CTSM_ROOT]$ ./manage_externals/checkout_externals` is included under both the ["user"](https://metos-uio.github.io/CTSM-Norway-Documentation/get/#how-to-get-ctsm-for-users) header and the ["developer" ](https://metos-uio.github.io/CTSM-Norway-Documentation/get/#how-to-get-ctsm-for-developers)header. Do we want to keep it that way (meaning there will be some repetition), or is it better to create one new header for the manage externals stuff?

- **For CTSM**: My thinking in doing it in this way is that if you are a user and "only", would like to run the model you would start from a specific version of a tag. On the other hand, if you are a developer the best practice would be to always branch off from the latest master. We could also put the developing part under its own file and include it here #18. 

- **For FATES**: It gets a bit more complicated to start working because you need a version of CTSM which is compatible with FATES. I think @huitang-earth has tried to give a description that separates this? 

- **The platform**: I think we have agreed to move the platform documentation to its own docs site (issued here #20, @lasseke will come back to this when they have had a platform discussion.)

In summary: 
I think it makes sense to emphasize the difference between starting from master or starting from a given version, but I agree that it is not organized in the best possible way right now and I do not have strong feelings about how we could make it better. If you have a better idea I think you should go ahead @ecaas :)

**Comment by sunnivin** [comment]
Thinking about it again I think I like the idea of including a new header for 'manage_externals' and then moving the developing stuff into #18 and in [this](https://metos-uio.github.io/CTSM-Norway-Documentation/develop/) file.

**Comment by sunnivin** [comment]
With @ecaas's contribution, I think the documentation is in a better place now. 

I will close this issue for now. Let's think about it again when we are doing #7.
